lib/network/encoding.go

```diff
@@ -192,6 +192,7 @@ func MarshalRegisteredType(data ProtocolMessage) ([]byte, error) {
 	var err error
 	if buf, err = protobuf.Encode(data); err != nil {
 		dbg.Error("Error for protobuf encoding:", err)
+		dbg.Error(dbg.Stack())
```

Comment:
Nice, this will help with some nasty protobuf errors. On the other hand, we should be careful not to output the stack if for instance someone sent a corrupt message on purpose or by accident.